packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1485,7 +1485,7 @@ class RenderDecoratedBox extends RenderProxyBox {
             'After painting it, the canvas save count was ${context.canvas.getSaveCount()}. '
             'Every call to save() or saveLayer() must be matched by a call to restore().\n'
             'The decoration was:\n'
-            '${decoration.toStringDeep()}\n'
+            '  $decoration\n'
```

Comment:
the regular toString for a decoration is now just fine. Having the method be toStringDeep for a decoration seems odd as a decoration doesn't have children.